Sources/Logging/LogHandler.swift

```diff
@@ -164,19 +159,17 @@ public protocol LogHandler: _SwiftLogSendableLogHandler {
 }
 
 extension LogHandler {
-    @available(*, deprecated, message: "You should implement this method instead of using the default implementation")
-    public func log(level: Logger.Level,
-                    message: Logger.Message,
-                    metadata: Logger.Metadata?,
-                    source: String,
-                    file: String,
-                    function: String,
-                    line: UInt) {
-        self.log(level: level, message: message, metadata: metadata, file: file, function: function, line: line)
-    }
-
+    
+    /// SwiftLog 1.0 compatibility method. Please do _not_ implement, implement
```

Comment:
this cannot be an extension method; this is a protocol requirement that existing `LogHandler` implementations implement; removing the requirement will not yield the expected behavior with such libraries.